geomesa-gs-monitor-elasticsearch/src/main/scala/org/geomesa/gs/monitor/elastic/ElasticRequestDataListener.scala

```diff
@@ -28,11 +29,12 @@ import java.util.function.BiFunction
 import scala.collection.JavaConverters._
 import scala.concurrent.TimeoutException
 
-class ElasticRequestDataListener extends RequestDataListener
+class ElasticRequestDataListener(catalog: Catalog) extends RequestDataListener
   with ApplicationListener[ApplicationEvent] with LazyLogging {
 
   import org.geomesa.gs.monitor.elastic.ElasticRequestDataListener._
 
+  ExtendedRequestData.catalog = catalog
```

Comment:
instead of having this be static (which is messy and hard to test), we should pass it in when constructing the `new ExtendedRequestData(rd)` down on line 85.